CharacterCreation/03_Feats.txt

```diff
@@ -525,42 +624,48 @@ Prerequisites:
 fighting technique.
 
 == Unlocked, Boomstick ==
+
 Prerequisites:
 
 * Must be a Gunslinger
 
-	Enables the use of shotguns by Gunslingers.
+	Enables the use of shotguns by the listed classes.

```

Comment:
It feels janky to use "the listed classes" for literally just one class. Is it cleaning up commit junk, because I deliberately changed this in PR#440